geniza/corpus/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class Library(models.Model):
+    '''Library or archive that holds Geniza fragments'''
+    name = models.CharField(max_length=255, unique=True)
+    abbrev = models.CharField('Abbreviation', max_length=255, unique=True)
+    url = models.URLField('URL', blank=True)
+
+    class Meta:
+        verbose_name_plural = 'Libraries'
+        ordering = ['name']
+
+    def __str__(self):
+        return self.name
```

Comment:
super minor (and maybe I missed discussion on this) but I wonder if it'd be helpful to have it be `self.name (self.abbrev)` so they look like `British Library (BL)`. that might at least help us get more familiar with the abbreviations since we'll see them more.

Reply:
oh, good point. if that's the case i agree it doesn't add much